packages/@react-aria/toast/docs/useToast.mdx

```diff
@@ -0,0 +1,452 @@
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/toast';
+import statelyDocs from 'docs:@react-stately/toast';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-aria/toast/package.json';
+import Anatomy from './toast-anatomy.svg';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import {Keyboard} from '@react-spectrum/text';
+
+```tsx import
+import {useToastState} from '@react-stately/toast';
+import {useToastRegion} from '@react-aria/toast';
+```
+
+---
+category: Status
+keywords: [toast, notifications, alert, aria]
+after_version: 3.0.0
+---
+
+# useToast
+
+<PageDescription>{docs.exports.useToast.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useToastRegion', 'useToast']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/WAI/ARIA/apg/patterns/alert/'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useToastRegion} links={docs.links} />
+<FunctionAPI function={docs.exports.useToast} links={docs.links} />
+
+## Features
+
+There is no built in way to toast notifications in HTML. <TypeLink links={docs.links} type={docs.exports.useToastRegion} /> and <TypeLink links={docs.links} type={docs.exports.useToast} /> help achieve accessible toasts that can be styled as needed.
+
+* **Accessible** – Toasts follow the [ARIA alert pattern](https://www.w3.org/WAI/ARIA/apg/patterns/alert/). They are rendered in a [landmark region](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/), which keyboard and screen reader users can easily jump to when an alert is announced.
+* **Focus management** – When a toast unmounts, focus is moved to the next toast if any. Otherwise, focus is restored to where it was before navigating to the toast region.
+* **Priority queue** – Toasts are displayed according to a priority queue, displaying a configurable number of toasts at a time. The queue can either be owned by a provider component, or global.
+* **Animations** – Toasts support optional entry and exit animations.
+
+## Anatomy
+
+<Anatomy />
+
+A toast region is an ARIA landmark region labeled "Notifications" by default. A toast region contains one or more visible toasts, in priority order. When the limit is reached, additional toasts are queued until the user dismisses one. Each toast is an ARIA alert element, containing the content of the notification and a close button.
+
+Landmark regions including the toast container can be navigated using the keyboard by pressing the <Keyboard>F6</Keyboard> key to move forward, and the <Keyboard>Shift</Keyboard> + <Keyboard>F6</Keyboard> key to move backward. This provides an easy way for keyboard users to jump to the toasts from anywhere in the app. When the last toast is closed, keyboard focus is restored.
```

Comment:
F6 cycles between landmarks, and in the docs there are no other landmarks than the toasts. I don't want to complicate the docs too much. When we document landmarks we can include a link maybe. Otherwise, I think the toast landmark alone is better than nothing - you can jump to the toasts easily, and if you close one, your focus is restored. As such, this isn't really a toast feature so much as a landmark one.